assets/airports/ksea.json

```diff
@@ -280,20 +644,185 @@
+            "destination": "",
+            "category": "departure",
+            "route": "KSEA.HAROB5.HISKU",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 8.75,
+            "airlines": [
+                ["aal", 4.05],
+                ["aca", 0.95],
+                ["asa", 30.86],
+                ["baw/long", 0.28],
+                ["cfs", 0.57],
+                ["cpz", 5.05],
+                ["dal", 9.80],
+                ["dlh/long", 0.17],
+                ["eva", 0.21],
+                ["fdx", 1.18],
+                ["fft", 0.39],
+                ["hal", 0.43],
+                ["jbu", 0.96],
+                ["kal/long", 0.28],
+                ["nks", 0.03],
+                ["qxe", 21.17],
+                ["skw", 10.24],
+                ["swa", 5.93],
+                ["uae", 0.41],
+                ["ual", 4.34],
+                ["vrd", 1.23]
+            ]
+        },{
+            "origin": "ksea",
+            "destination": "",
+            "category": "departure",
+            "route": "KSEA.HAROB5.HQM",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 8.75,
+            "airlines": [
+                ["aal", 4.05],
+                ["aca", 0.95],
+                ["asa", 30.86],
+                ["baw/long", 0.28],
+                ["cfs", 0.57],
+                ["cpz", 5.05],
+                ["dal", 9.80],
+                ["dlh/long", 0.17],
+                ["eva", 0.21],
+                ["fdx", 1.18],
+                ["fft", 0.39],
+                ["hal", 0.43],
+                ["jbu", 0.96],
+                ["kal/long", 0.28],
+                ["nks", 0.03],
+                ["qxe", 21.17],
+                ["skw", 10.24],
+                ["swa", 5.93],
+                ["uae", 0.41],
+                ["ual", 4.34],
+                ["vrd", 1.23]
+            ]
+        },{
+            "origin": "ksea",
+            "destination": "",
+            "category": "departure",
+            "route": "KSEA.SUMMA9.BKE",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 22.5,
+            "airlines": [
+                ["aal", 4.05],
+                ["aca", 0.95],
+                ["asa", 30.86],
+                ["baw/long", 0.28],
+                ["cfs", 0.57],
+                ["cpz", 5.05],
+                ["dal", 9.80],
+                ["dlh/long", 0.17],
+                ["eva", 0.21],
+                ["fdx", 1.18],
+                ["fft", 0.39],
+                ["hal", 0.43],
+                ["jbu", 0.96],
+                ["kal/long", 0.28],
+                ["nks", 0.03],
+                ["qxe", 21.17],
+                ["skw", 10.24],
+                ["swa", 5.93],
+                ["uae", 0.41],
+                ["ual", 4.34],
+                ["vrd", 1.23]
+            ]
+        },{
```

Comment:
What do you want do with this? There needs to be a convention that goes throughout the file, do you want to do `}, {` or 
`},`
`{`?